src/Terminus.php

```diff
@@ -163,8 +164,10 @@ private function addPluginsCommandsAndHooks()
         $discovery = $this->getContainer()->get(PluginDiscovery::class, [$this->getConfig()->get('plugins_dir'),]);
         $plugins = $discovery->discover();
         $version = $this->config->get('version');
+        $classLoader = $this->getContainer()->get('pluginClassLoader');
```

Comment:
Directly accessing the container again. I could follow the advice I gave above, and just instantiate the classloader once here.